### PR TITLE
makes plasmacutters cost less to recharge

### PR DESCRIPTION
**Pull request by Serxule**
# About The Pull Request

simple PR changing some numbers to make plasma cutters cost less plasma to recharge

## Why It's Good For The Game

plasma cutters currently cost a lot of plasma to recharge, this makes them actually useful by using less plasma. 

## Changelog

:cl:
balance: Plasma Cutters require less plasma to charge

/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/energy/special.dm [hunk 1]
```diff
@@ -165,15 +165,15 @@
 /obj/item/gun/energy/plasmacutter/attackby(obj/item/I, mob/user)
 	var/charge_multiplier = 0 //2 = Refined stack, 1 = Ore
 	if(istype(I, /obj/item/stack/sheet/mineral/plasma))
-		charge_multiplier = 2
+		charge_multiplier = 5
```
Comment (nednaZ) [inline code/modules/projectiles/guns/energy/special.dm:168]:
```suggestion
		charge_multiplier = 5 // MonkeStation Edit
```

File: code/modules/projectiles/guns/energy/special.dm [hunk 2]
```diff
@@ -165,15 +165,15 @@
...
 	if(istype(I, /obj/item/stack/ore/plasma))
-		charge_multiplier = 1
+		charge_multiplier = 3
```
Comment (nednaZ) [inline code/modules/projectiles/guns/energy/special.dm:170]:
```suggestion
		charge_multiplier = 3 //MonkeStation Edit
```

File: code/modules/projectiles/guns/energy/special.dm [hunk 3]
```diff
@@ -165,15 +165,15 @@
...
 	if(charge_multiplier)
 		if(cell.charge == cell.maxcharge)
 			to_chat(user, "<span class='notice'>You try to insert [I] into [src], but it's fully charged.</span>") //my cell is round and full
 			return
 		I.use(1)
-		cell.give(50*charge_multiplier)
+		cell.give(100*charge_multiplier)
```
Comment (nednaZ) [inline code/modules/projectiles/guns/energy/special.dm:176]:
```suggestion
		cell.give(100*charge_multiplier) //MonkeStation Edit
```

**Review by nednaZ — CHANGES_REQUESTED** [review]
Numbers look fine to me, the plasma cutter has always been a pain to actually charge up.
Just need the code comments to mark out that we have edited these sections locally.

**Comment by Serxule** [comment]
didnt realize that, done

**Comment by nednaZ** [comment]
Just leaving for reference.
1 ore is 30% charge on a regular cutter.
1 sheet is 50% charge on a regular cutter.

**Pull request merged**